feat(lb): add cockpit metrics doc (#2016)
scaleway/docs-content: network/load-balancer/how-to/monitor-lb-cockpit.mdx

---
meta:
  title: How to monitor your Load Balancer with Scaleway Cockpit 
  description: This page explains how to monitor your Load Balancer with Scaleway Cockpit 
content:
  h1: How to monitor your Load Balancer with Scaleway Cockpit 
  paragraph: This page explains how to monitor your Load Balancer with Scaleway Cockpit
tags: load-balancer cockpit observability dashboard metrics logs alerts 
dates:
  validation: 2023-08-07

>   posted: 2023-01-24

Reviewer: ```suggestion
  posted: 2023-08-07
```

categories:
  - network
---

You can view your Load Balancer's metrics by using [Scaleway Cockpit](/observability/cockpit/quickstart/). Load Balancer is fully integrated into Cockpit, and allows you to monitor your Load Balancers frontends, backends and backend servers at a glance, and visualize your metrics and traffic. This page explains how to get started with Scaleway Cockpit for viewing your Load Balancers metrics.

<Macro id="iam-requirements" />
 
<Message type="requirement">
  - You have an account and are logged into the [Scaleway console](https://console.scaleway.com) 
  - You have [created a Load Balancer](/network/load-balancer/how-to/create-load-balancer)
</Message>

## How to activate your Cockpit

Scaleway Cockpit is deactivated by default, but you can easily activate it in the Scaleway console by following the procedure detailed [here](/observability/cockpit/how-to/activate-cockpit/).

> ## How to access the Grafana dashboard.

Reviewer 2: ```suggestion
## How to access the Grafana dashboard
```

To view your Load Balancer's metrics, use the Grafana dashboard which is accessible from the Scaleway console:
  - First you must create a user and their associated credentials (a username and password) for Grafana
  - Then, you can use these credentials to access the Grafana dashboard for all your Scaleway products, via the Scaleway console.

> 1. Create a Grafana user and retrieve their username and password by [following this procedure](https://www.scaleway.com/en/docs/observability/cockpit/how-to/retrieve-grafana-credentials/).

Reviewer 2: ```suggestion
1. Create a Grafana user and retrieve their username and password by [following this procedure](/observability/cockpit/how-to/retrieve-grafana-credentials/).
```

2. From the Scaleway console [Cockpit overview page](https://console.scaleway.com/cockpit/overview), click **Open dashboards** in the top right corner.

3. Enter your Grafana username and password to log in.

    You are directed to the list of your Scaleway Cockpit dashboards.

    <Lightbox src="scaleway-grafana-dashboard.webp" alt="" />

4. Click **LB Overview** in the list of dashhboards on the left.

    You are directed to the Load Balancer dashboard.

    <Lightbox src="scaleway-lb-dashboard.webp" alt="" />

## How to understand your Load Balancer metrics

### Choosing metric parameters

At the top of the dashboard, you can configure the following parameters, which control which metrics are displayed:

    <Lightbox src="scaleway-loadbalancer-params.webp" alt="" />

> - **A: Time period**: Click this dropdown to configure the time period you want the metrics display to cover. You can set an absolute time range, from one fixed date-time to antoher, or use a quick range such as `Last 15 minutes` or `Last 3 hours`.

Reviewer: ```suggestion
- **A: Time period**: Click this dropdown to configure the time period you want the metrics displayed to cover. You can set an absolute time range, from one fixed date-time to another, or use a quick range such as `Last 15 minutes` or `Last 3 hours`.
```

- **B: Refresh rate**: Use the arrow icon to refresh the dashboard, and/or use the dropdown next to it to set the automatic refresh rate.
- **C: Load Balancer name**: Click this dropdown to select which of your Load Balancers you want to display metrics for. 
- **D: Frontend name**: Click this dropdown to select which of your Load Balancer's frontends you want to display metics for. You can select all attached frontends, or limit the display to only certain frontends.
- **E: Backend name**: Click this dropdown to select which of your Load Balancer's backends you want to display metics for. You can select all attached backends, or limit the display to only certain backends.

### LoadBalancer status

In this section, you see various graphs which report on the status of different components of your Load Balancer.

    <Lightbox src="scaleway-loadbalancer-status.webp" alt="" />

> - **Healthy loadbalancer backends**: Shows how many of your Load Balancer's backends currently have a healthy status. A backend is considered healthy if it at least one of its [backend servers](/network/load-balancer/concepts/#backend-servers) is healthy and up.

Reviewer: ```suggestion
- **Healthy loadbalancer backends**: Shows how many of your Load Balancer's backends currently have a healthy status. A backend is considered healthy if at least one of its [backend servers](/network/load-balancer/concepts/#backend-servers) is healthy and up.
```
